charts/caddy-ingress-controller/values.schema.json

```diff
@@ -174,6 +174,71 @@
         }
       }
     },
+    "service": {
+      "$id": "#/properties/service",
+      "type": "object",
+      "required": [
+        "type",
+        "ipDualStack"
+      ],
+      "properties": {
+        "internalTrafficPolicy": {
+          "$id": "#/properties/service/properties/internalTrafficPolicy",
+          "type": "string",
+          "enum": [
+            "Cluster",
+            "Local"
+          ]
+        },
+        "externalTrafficPolicy": {
+          "$id": "#/properties/service/properties/externalTrafficPolicy",
+          "type": "string",
+          "enum": [
+            "Cluster",
+            "Local"
+          ]
+        },
+        "annotations": {
+          "$id": "#/properties/service/properties/annotations",
+          "type": "object"
+        },
+        "type": {
+          "$id": "#/properties/service/properties/type",
+          "type": "string",
+          "enum": [
+            "ClusterIP",
+            "NodePort",
+            "LoadBalancer",
+            "ExternalName"
+          ]
+        },
+        "clusterIP": {
+          "$id": "#/properties/service/properties/clusterIP",
+          "type": "string"
+        },
+        "ipDualStack": {
+          "$id": "#/properties/service/properties/name",
+          "type": "object",
+          "required": [
+            "enabled"
+          ],
+          "properties": {
+            "enabled": {
+              "$id": "#/properties/service/properties/ipDualStack/properties/enabled",
+              "type": "boolean"
+            },
+            "ipFamilies": {
+              "$id": "#/properties/service/properties/ipDualStack/properties/ipFamilies",
+              "type": "array"
+            },
```

Comment:
We can make the array allow only items in the `IPv4` and `IPv6` enum here, to prevent possible config mistake, WDYT?

Reply:
sounds good